Source/DafnyCore/Generic/ReporterExtensions.cs

```diff
@@ -46,6 +46,7 @@ public static class ErrorReporterExtensions {
   private const string RelatedLocationCategory = "Related location";
   public const string RelatedLocationMessage = RelatedLocationCategory;
   private const string RelatedMessageCategory = "Related message";
+  public const string AssertedExprCategory = "Asserted expression";
```

Comment:
I love that this fits in nicely with the existing reporter infrastructure!

Reply:
Indeed - for a while I thought I might need to make changes in Boogie itself in order to propagate the necessary expressions, but this worked out nicely.